src/extensions/lens-extension.ts

```diff
@@ -73,23 +81,20 @@ export class LensExtension {
     return FilesystemProvisionerStore.getInstance().requestDirectory(this.id);
   }
 
-  get description() {
-    return this.manifest.description;
-  }
-
   @action
   async enable() {
     if (this.isEnabled) return;
     this.isEnabled = true;
-    this.onActivate();
+    this.onActivate?.();
```

Comment:
why `?.()` is needed?

Reply:
Haha oops, I had thought about making those methods be optional fields instead but then reverted it. I guess I forgot to revert this too